transfer-ownership/src/agent.spec.js

```diff
@@ -0,0 +1,58 @@
+const {
+    TransactionEvent, FindingType, FindingSeverity, Finding,
+  } = require('forta-agent');
+const ethers = require('ethers');
+const { handleTransaction } = require('./agent');
+
+//Take same list of monitored events as used in the agent script
+const ABI = require("./events.json");
```

Comment:
Suggest changing ABI to abi and EVENTS to events just for consistency...we've been using all caps only for hardcoded basic types - integers and strings.